Description: add MIPS-specific stub prototype for build log scanner
Author: mirabilos <tg@debian.org>

--- a/include/sys/tls.h
+++ b/include/sys/tls.h
@@ -71,6 +71,10 @@ int arch_prctl(unsigned int what, void*
 
 #define __ABI_TLS_REGISTER	"%g7"
 
+#elif defined(__mips__)
+
+int set_thread_area(void *);
+
 #else
 
 #warning "need proper sys/tls.h for this platform"
